statsmodels/multivariate/tests/Bartlett-Sphericity.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Sep 10 20:21:04 2016
+
+Bartlett Sphericity Test Function:
+
+- This test evaluates sampling adequacy for exploratory Factor Analysis
+
+Bartlett_Sphericity function has two inputs:
+
+- The Dataset (numerical or ordinal variables only)
+- The correlation method (spearman or pearson)
+
+It Outputs the test result, degrees of freedom and p-value
+
+
+@authors: Rui Sarmento
+          Vera Costa
+"""
+
+#Bartlett Sphericity Test
+
+def Bartlett_Sphericity(dataset, corr_method="pearson"):
+    
+    r"""
+    
+    Parameters
+    ----------
+    dataset : dataframe, mandatory (numerical or ordinal variables)
+        
+    corr_method : {'pearson', 'spearman'}, optional
+        
+    Returns
+    -------
+    out : str
+        The function outputs the test value (chi2), the degrees of freedom (ddl)
+        and the p-value.
+        It also delivers the n_p_ratio if the number of instances divided 
+        by the numbers of variables is more than 5
+        
+        Ex:
+        chi2:  410.27280642443156
+        ddl:  45.0
+        p-value:  8.73359410503e-61
+        n_p_ratio:    20.00
+    
+    References
+    ----------
+    [1] R. Sarmento and V. Costa,
+    "Comparative Approaches to Using R and Python for Statistical Data Analysis"
+    in press, Cybertech Publishing, 2016.

```

Comment:
That doesn't sound like a reference that is specifically relevant for explaining the background of the function